lib/ansible/modules/system/systemd-modules-load.py

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2018, Susant Sahani <ssahani@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['stableinterface'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: systemd-modules-load
+short_description:  systemd-modules-load
+description:
+    - Allows you to  generate systemd-modules-load configuration files.
+version_added: "2.8"
+options:
+    conf_path:
+        description:
+            - Specifies the path where to write the configuration files.
+        default: "/etc/modules-load.d"
+        choices: [ "/etc/modules-load.d", "/run/modules-load.d", "/usr/lib/modules-load.d" ]
+    file_name:
+        description:
+            - This configuration file name where the configurations will be written. Note the file name will be
+              automatically have the extension .conf.
+    modules:
+        description:
+            - Space separated list of module names.
+    state:
+        description:
+            - Whether configuration files should be added or absentd.
+        choices: [ "present", "absent" ]
+author: "Susant Sahani (@ssahani) <ssahani@gmail.com>"
+'''
+
+EXAMPLES = '''
+# Create config file
+- systemd-modules-load:
+       file_name: my_tunnel
+       modules:  ipip sit vti
+       state: present
+
+# Remove config file
```

Comment:
use` name: ` in task instead of comment, so it can be displayed in all documentation forms